datasource/index.js

```diff
@@ -218,13 +217,13 @@ module.exports = yeoman.generators.Base.extend({
         name: 'installConnector',
         message: 'Install ' + npmModule,
         type: 'confirm',
-        default: true
-      }
+        default: true,
+      },
     ];
 
     this.prompt(prompts, function(props) {
       if (props.installConnector) {
-        this.npmInstall([npmModule], {'save': true});
+        this.npmInstall([npmModule], { 'save': true });
```

Comment:
IIRC, the latest published version of `eslint-config-loopback` changes the rule about curly-spacing to `{'save':true}`.